themes/default/Post.template.php

```diff
@@ -476,6 +476,7 @@ function template_add_new_attachments()
 		numOfAttachmentAllowed: ' . $context['attachments']['num_allowed'] . ',
 		totalAttachSizeUploaded: ' . (isset($context['attachments']['total_size']) && !empty($context['attachments']['total_size']) ? $context['attachments']['total_size'] : 0) . ',
 		numAttachUploaded: ' . (isset($context['attachments']['quantity']) && !empty($context['attachments']['quantity']) ? $context['attachments']['quantity'] : 0) . ',
+		fileDisplayTemplate: \'<div class="statusbar"><div class="info"></div><div class="progressBar"><div></div></div><div class="control fa fa-times-circle"></div></div>\',
```

Comment:
Done (mostly) https://github.com/elkarte/Elkarte/pull/2820/commits/f15862286ad201c211c95a32a8bdb6556ac03c8f
There is only the http://fontawesome.io/icon/share-alt-square/ left, not sure what I can use in its place.